app/controllers/users/passwords_controller.rb

```diff
@@ -14,14 +14,14 @@ def create
     end
 
     if user.prevent_password_resets?
-      flash[:error] = t(".reset_blocked", contact_abuse_link: view_context.link_to(t(".contact_abuse"), new_abuse_report_path)).html_safe
+      flash[:error] = t(".reset_blocked_html", contact_abuse_link: view_context.link_to(t("users.contact_abuse"), new_abuse_report_path))
```

Comment:
Apologies for coming back to this after already approving, but I had some discussion with the Translation team:

It would be nice if `reset_blocked` would continue to use a different locale key for the policy and abuse link text than the status texts. That allows translators to translate the link differently in context, since the sentence in `reset_blocked` is different than the one used in `suspension_notice_html` and `ban_notice_html`.

```suggestion
      flash[:error] = t(".reset_blocked_html", contact_abuse_link: view_context.link_to(t(".contact_abuse"), new_abuse_report_path))
```

Reply:
done